components/bigquery-components/pyproject.toml

```diff
@@ -1,30 +1,30 @@
-[project]
+[tool.poetry]
 name = "bigquery-components"
-authors = [
-    {name = "Example User", email = "user@example.com"},
-]
-description = "BigQuery components"
+version = "0.1.0"
+authors = ["roberta nwokonko <roberta.nwokonko@datatonic.com>"]
+description = "BigQuery components created by Datatonic"
```

Comment:
remove references to Datatonic - instead something like "BigQuery components for Vertex Pipelines"

Reply:
265777f